docs/source/use_cases.rst

```diff
@@ -0,0 +1,12 @@
+.. _use-cases:
+
+Use Cases
+=========
+
+The Data Resource Generator (DRG) allows you to dynamically generate a database and API with a configuration file. The biggest use cases for the DRG in general are automation and data collaboration.
+
+#. This can be used to allow non-database users to create and use databases with a RESTful API interface
+#. Quickly stand up a database with RESTful API programmatically.
+#. By leveraging Open API Specification 3.0 (OAS3 or Swagger) you can automate the building of API clients.
+#. Core parts of the library can be used to automate database generation.
+#. You can share your configuration file to communicate with partners what types of data you have.
```

Comment:
I'd like to see these statements embrace grammatical parallelism, i.e., have the same structure. What about something where all bullet points start with an imperative verb?

> #. Quickly and programmatically stand up a database with a RESTful API.
#. Allow non-database admins to interact with databases via a RESTful API.
#. Secure and protect all, some, or no API endpoints.
#. Leverage the Open API Specification 3.0 (OAS3 or Swagger) to automate the building of API clients.
#. Share your configuration file to communicate with partners what types of data you have.
#. Automate database generation (without an API) using core parts of the DRG library.

Reply:
Excellent!